flx-rs.el

```diff
@@ -34,12 +34,18 @@
 (require 'cl-lib)
 (require 'subr-x)
 
-(defconst flx-rs--bin-dir
+(defgroup flx-rs nil
+  "flx in Rust using dynamic module"
+  :prefix "flx-rs--")
+
+(defcustom flx-rs--bin-dir
   (concat (file-name-directory load-file-name) "bin/")
-  "Pre-built binaries directory path.")
+  "Pre-built binaries directory path."
+  :type 'directory)
 
-(defconst flx-rs--dyn-name "flx_rs_core"
-  "Dynamic module name.")
+(defcustom flx-rs--dyn-name "flx_rs_core"
+  "Dynamic module name."
+  :type 'string)
```

Comment:
Can you update these variables without the extra dash from the prefix. The two dashes `--` are for private variables, and since these became public, we want it to change to `-`, one dash only.